uniprot-datastore/src/main/java/org/uniprot/store/datastore/voldemort/uniprot/VoldemortRemoteCachingUniProtKBEntryStore.java

```diff
@@ -0,0 +1,56 @@
+package org.uniprot.store.datastore.voldemort.uniprot;
+
+import org.uniprot.core.json.parser.uniprot.UniprotJsonConfig;
+import org.uniprot.core.uniprot.UniProtEntry;
+import org.uniprot.core.uniprot.UniProtEntryType;
+import org.uniprot.store.datastore.voldemort.VoldemortRemoteCachingJsonBinaryStore;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+
+/**
+ * This class contains methods to save UniProtKB entries to a remote Voldemort store, whilst caching
+ * Swiss-Prot entries locally.
+ *
+ * <p>Created 27/02/2020
+ *
+ * @author Edd
+ */
+public class VoldemortRemoteCachingUniProtKBEntryStore
```

Comment:
do you think we can reuse the existing VoldemortRemoteUniProtKBEntryStore and do not create an extra one?